src/components/WalletDropdown/SlideOutMenu.tsx

```diff
@@ -0,0 +1,62 @@
+import { ChevronLeft } from 'react-feather'
+import { Text } from 'rebass'
+import styled, { useTheme } from 'styled-components/macro'
+
+const BackSection = styled.div`
+  position: relative;
+  display: flex;
+  padding: 0 16px;
+  color: ${({ theme }) => theme.textTertiary};
+  cursor: default;
+  :hover {
+    text-decoration: none;
+  }
+`
+
+const Menu = styled.div`
+  width: 100%;
+  height: 100%;
+  font-size: 16px;
+  overflow-y: scroll;
+  &::-webkit-scrollbar {
```

Comment:
rm webkit specific

Reply:
unless there's a specific reason to keep this, since we don't really do this elsewhere?